iosApp/Podfile

```diff
@@ -1,3 +1,4 @@
+source 'https://cdn.cocoapods.org/'
```

Comment:
I don't think it should be necessary to `source` the CocoaPods CDN. The official CocoaPods source is implicit unless another source is also used, in which case it must then be added as well.